include/drivers/led_display.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include "drivers/gpio_pin.hpp"
+#include "microbit_v2.h"
+
+namespace edge::drivers {
+
+// Synchronous
```

Comment:
theres gotta be a way to enforce that its sync wtihout having to comment synchronous above it

Reply:
solution is prolly to write a driver interface or something

Reply:
closing bc we half agreed